models/intel/index.md

```diff
@@ -193,6 +194,10 @@ Deep Learning compressed models
 | [mobilenetv2-int8-tf-0001](./mobilenetv2-int8-tf-0001/description/mobilenetv2-int8-tf-0001.md)                                                 | 0.615                | 3.489      |
 | [mobilenetv2-int8-sparse-v1-tf-0001](./mobilenetv2-int8-sparse-v1-tf-0001/description/mobilenetv2-int8-sparse-v1-tf-0001.md)                   | 0.615                | 3.489      |
 | [mobilenetv2-int8-sparse-v2-tf-0001](./mobilenetv2-int8-sparse-v2-tf-0001/description/mobilenetv2-int8-sparse-v2-tf-0001.md)                   | 0.615                | 3.489      |
+| [icnet-camvid-int8-onnx-0001](./icnet-camvid-int8-0001/description/icnet-camvid-int8-onnx-0001.md)                   | -                | -      |
```

Comment:
onnx missed in directory name, should be:
`./icnet-camvid-int8-onnx-0001/description/icnet-camvid-int8-onnx-0001.md`